src/fc/fc_domain.c

```diff
@@ -392,6 +432,15 @@ fcConfigAndStartVM(virFCDriver *driver G_GNUC_UNUSED,
         return -1;
     }
 
+    for (i = 0; i < vm->def->nnets; ++i) {
```

Comment:
does the formatting test complain if we declare the `i` variable here?

Reply:
Sadly no, it has a check for not declaring variables in a loop